internal/resources/oncall/resource_integration.go

```diff
@@ -218,7 +222,7 @@ func ResourceIntegration() *schema.Resource {
 						return false
 					}
 					for k, v := range oldTemplateMap {
-						if newTemplateMap[k] != v {
+						if fmt.Sprintf("%v", newTemplateMap[k]) != fmt.Sprintf("%v", v) {
```

Comment:
This is a bit convoluted. I'm approving because I eventually understood it, but it took me a short while.